plugin-server/src/main/ingestion-queues/kafka-queue.ts

```diff
@@ -220,7 +112,18 @@ export class KafkaQueue implements Queue {
         return Promise.resolve()
     }
 
-    resume(targetTopic: string = this.pluginsServer.KAFKA_CONSUMPTION_TOPIC!, partition?: number): void {
+    async sleep(sleepMs: number, partition: number, topic = this.bufferTopic): Promise<void> {
```

Comment:
Same as below - does the topic default here even work? Why do we need this argument?